chromadb/db/system.py

```diff
@@ -54,6 +55,12 @@ def delete_segment(self, id: UUID) -> None:
         """Create a new segment in the System database."""
         pass
 
+    @abstractmethod
+    def optimize_system(self) -> SystemOptimizationStats:
```

Comment:
Differentiating naming is needed as inheritance at SqliteDB overrides this method by the one defined in Producer.